app/assets/stylesheets/courses.scss

```diff
@@ -3,6 +3,5 @@
 // You can use Sass (SCSS) here: https://sass-lang.com/
 
 .tab-content {
-	max-height:270px;
```

Comment:
This is required for one of our stories. The filelist in the course overview must be limited.
Instead if deleting this CSS (and leaving the now useless overflow attribute) Please just switch the selector from `.tab-content` to `#fileTabsContent`
https://github.com/hpi-swt2/lecture-portal/blob/dev/app/views/courses/show.html.erb#L30

Reply:
And sorry from Team RESTless for breaking the system

Reply:
```suggestion
	max-height:270px;
```

Reply:
solved in #212

Reply:
use `git revert origin/dev app/assets/stylesheets/courses.scss` to reset this file to the dev branch state